README.md

```diff
@@ -143,13 +135,13 @@ If you want to disable the error reporting you can do this by setting the state
 ### Notification System
 **Feature status:** Technology preview since js-controller 3.2.0
 
-The notification system in ioBroker allows to set, detect and store notifications per Host and allows to query the details.
+The notification system in ioBroker allows setting, detect and store notifications per Host and allows to query the details.
```

Comment:
then we would also need to change to `detecting and storing`

Reply:
The notification system in ioBroker allows setting, detecting and storing notifications ...